app/assets/javascripts/controllers/CommonChatController.js

```diff
@@ -181,7 +189,8 @@ export default class CommonChatController {
     closeChat() {
 
         if (document.body.contains(document.getElementById("postChatSurveyWrapper"))) {
-            this._sendPostChatSurveyWebchat(this.sdk).closePostChatSurvey(automaton, timestamp);
+            //this._sendPostChatSurveyWebchat(this.sdk).closePostChatSurvey(automaton, timestamp)
```

Comment:
Can we get rid of commented code?

Reply:
No, will make DEP-2480 easier as we need both methods for that ticket